_workinggroups/detsim.md

```diff
@@ -16,4 +16,7 @@ technologies) are an important area of work.
 
 [Mandate](/organization/working-group-mandates.html).
 
-Convenors: Gloria Corti (LHCb), Heather Gray (ATLAS), Witek Pokorski (Geant4)
+The group's meeting page is [in Indico](https://indico.cern.ch/category/10916/) and
+discussions are held on the [hsf-simulation](https://groups.google.com/forum/#%21forum/hsf-simulation) list.
+
+Convenors: Gloria Corti (LHCb) (mailto:Gloria.Corti@cern.ch), Heather Gray (ATLAS) (mailto:heather.gray@cern.ch), Witek Pokorski (Geant4) (mailto:Witold.Pokorski@cern.ch)
```

Comment:
Did you want these to be links or copy/paste addresses? If you want a link use <>s, e.g.,

<mailto:heather.gray@cern.ch>

Reply:
Yes, I want the <>. Updating.